foreshadow/tests/test_core/test_cache_manager.py

```diff
@@ -236,7 +236,11 @@ def test_cache_manager_dict_serialize(store):
         else:
             expected["store"][key] = PrettyDefaultDict(lambda: None)
 
-    assert expected == cs.dict_serialize(deep=True)
+    # TODO it's an ugly fix. Idealy we should add the default config section
+    #  back but it will make the json file really bulky.
+    cs_serialized = cs.dict_serialize(deep=True)
+    cs_serialized["store"].pop("config")
+    assert expected == cs_serialized
```

Comment:
Please also check the config object popped from the dictionary.

Reply:
Done.